examples/objectStorage.ts

```diff
@@ -0,0 +1,21 @@
+import { Handler } from "../src";
+
+export const handler: Handler.ObjectStorage = (event) => {
+    for (const message of event.messages) {
+        switch (message.event_metadata.event_type) {
+            case "yandex.cloud.events.storage.ObjectCreate":
```

Comment:
This could be enum? Like 

```
enum ObjectStorageEventType {
  Create = 'yandex.cloud.events.storage.ObjectCreate',
  ....
}
```

Reply:
This would require a centralized map of enums so they can be exported